misskey-websocket/src/broker.rs

```diff
@@ -179,8 +180,9 @@ impl Default for ReconnectConfig {
 
 impl Broker {
     pub async fn spawn(
-        url: Url,
+        uri: Uri,
         reconnect: ReconnectConfig,
+        additional_headers: HeaderMap,
```

Comment:
`additional_headers` comes before `reconnect` feels natural to me (but current one is ok to me)